tests/AppBundle/EventListener/CommentNotificationListenerTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\AppBundle\EventListener;
+
+use AppBundle\Entity\Comment;
+use AppBundle\Entity\Post;
+use AppBundle\Entity\User;
+use AppBundle\EventListener\CommentNotificationListener;
+use Prophecy\Argument;
+use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Unit test for the event listener.
+ *
+ * See http://symfony.com/doc/current/book/testing.html#unit-tests
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ *
+ *     $ cd your-symfony-project/
+ *     $ ./vendor/bin/phpunit
+ *
+ * @author Michael COULLERET <michael.coulleret@gmail.com>
+ */
+class CommentNotificationListenerTest extends \PHPUnit_Framework_TestCase
+{
+    public function testListenerOnCommentCreated()
```

Comment:
I'd split this one big test into 4 ones:

1. It should be subscribed to the comment creation event;
2. It should generate a link to created comment;
3. It should send a email to admin;
4. Is should translate a sending message.